test/specs/reference/tests/12.reference_values_edit.js

```diff
@@ -330,6 +330,7 @@ exports.execute = function (options) {
                     var jsonb=JSON.stringify(expectedValues[i].jsonb_col,undefined,2);
                     expect(values[1]).toBe(json);
                     expect(values[2]).toBe(jsonb);
+                    expect(values[3]).toBe(expectedValues[i].json_col_without_markdownpattern);
```

Comment:
Isn't the name of column `json_col_with_markdownpattern`? It shouldn't matter here since it's just a object that you're using internally for tests, but it's confusing that it says "without_markdown" while the column is with markdown.